contracts/modules/legacy/LegacyPooledStaking.sol

```diff
@@ -44,10 +44,7 @@ contract LegacyPooledStaking is IPooledStaking, MasterAware {
 
   /* Storage variables */
 
-  bool public initialized;
-
-  INXMToken public token;
-  ITokenController public tokenController;
+  ITokenController public _unused0;
```

Comment:
`token` and `initialized` were removed - let's leave a comment of what happens with the storage layout here as it's not obvious why this is OK